e2e/cypress/integration/messaging/message_reaction_gm_spec.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod @smoke
```

Comment:
Will do. Is there something in Jira that says what the Group is supposed to be for a given test? I hadn't really paid attention to this heading before since I've just been copying it between files.